twake/backend/node/src/services/user/entities/company_user.ts

```diff
@@ -0,0 +1,48 @@
+import { Column, Entity } from "../../../core/platform/services/database/services/orm/decorators";
+
+// backward compatibility with PHP where companies used to be `group`
+export const TYPE = "group_user";
+
+/**
+ * Link between a company and a user
+ */
+@Entity(TYPE, {
+  primaryKey: [["group_id"], ["user_id", "id"]],
+  type: TYPE,
+})
```

Comment:
Here we'll have an issue when we'll want to get all companies of one user. For now it is okay, but we'll need to create an other table or store companies id directly in user entity.

Reply:
And by the way: `[["group_id"], ["user_id", "id"]],` => `[["group_id"], "user_id", "id"],`

Reply:
Yes!